obs-packaging/kata-containers-image/build_image.sh

```diff
@@ -105,7 +105,7 @@ main(){
 	# Agent version
 	[ -n "${agent_version}" ] || agent_version="${kata_version}"
 
-	shift $(( "$OPTIND" - 1 ))
+	shift "$(( $OPTIND - 1 ))"
```

Comment:
This isn't mentioned in the commit - was the change necessary to make this work for you, or is it more a stylistic modification?

Reply:
... and I've just seen #110 :) Please could you remove this change from this PR and do all the `OPTIND` changes on #110 for clarity.